packages/flutter/lib/src/widgets/modal_barrier.dart

```diff
@@ -214,6 +337,17 @@ class AnimatedModalBarrier extends AnimatedWidget {
   /// {@macro flutter.widgets.ModalBarrier.onDismiss}
   final VoidCallback? onDismiss;
 
+  /// {@macro flutter.widgets.ModalBarrier.clipDetailsNotifier}
+  final ValueNotifier<EdgeInsets>? clipDetailsNotifier;
+
+  /// This hint text instructs user what they are able to do when they tap on
+  /// the [ModalBarrier], annouced in the format of 'Double tap to
+  /// $[semanticsOnTapHint].'
```

Comment:
this probably won't work, maybe give a example when `semanticsOnTapHint = 'bottom sheet' `